scripts/deploy.ts

```diff
@@ -431,6 +434,14 @@ function buildCommonPost(buildObj: BuildObj) {
     mylog('\n*** Upload to App Center Complete ***')
   }
 
+  if (zealotApiToken != null && zealotUrl != null && zealotChannelKey != null && !maestroBuild) {
+    mylog(`\n\nUploading to Zealot: ${zealotUrl}`)
+    mylog('***********************************************************************\n')
+
+    call(`curl -X POST "${zealotUrl}/api/apps/upload?token=${zealotApiToken}&channel_key=${zealotChannelKey}" -F "file=@${buildObj.ipaFile}"`)
+    mylog('\n*** Upload to Zealot Complete ***')
```

Comment:
Is `buildObj.ipaFile` the for Android _and_ iOS? Or is one platform not supported yet?